test/unit/tablestest/test_hdfstorage.py

```diff
@@ -247,11 +247,16 @@ def testStringCol(self):
         cols = [omero.columns.StringColumnI("name", "description", 16, None)]
         hdf.initialize(cols)
         cols[0].settable(hdf._HdfStorage__mea)  # Needed for size
-        cols[0].values = ["foo"]
+        cols[0].values = ["foo", "αbc"]
```

Comment:
Nitpicky suggestion: use characters that are not a homoglyph so it's clear to others?
```suggestion
        cols[0].values = ["foo", "მიკროსკოპის პონი"]
```

Reply:
I think we should add `მიკროსკოპის პონი` as a constant to `omero/Constants.ice`.